gluten-data/src/main/java/io/glutenproject/columnarbatch/ArrowColumnarBatches.java

```diff
@@ -66,12 +66,8 @@ private static void transferVectors(ColumnarBatch from, ColumnarBatch target) {
 
   public static ColumnarBatch load(BufferAllocator allocator, ColumnarBatch input) {
     if (!GlutenColumnarBatches.isIntermediateColumnarBatch(input)) {
-      throw new IllegalArgumentException("input is not intermediate Gluten columnar input. " +
-          "Please consider to use vanilla spark's row based input by setting one of the below" +
-          " configs: \n" +
-          "spark.sql.parquet.enableVectorizedReader=false\n" +
-          "spark.sql.inMemoryColumnarStorage.enableVectorizedReader=false\n" +
-          "spark.sql.orc.enableVectorizedReader=false\n");
+      // If the input is already spark ColumnarBatch, no need to load and directly return.
+      return input;
```

Comment:
Would this cause failure to case like `SparkColumnarFormat -> VeloxColumnarFormat`？And I notice we already have some handling at [GlutenPlugin](https://github.com/oap-project/gluten/blob/main/gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala#L62-L72) to set the three configs to false, but looks in some unit tests, they do not take effect.


From my try out, rewrite these tests to sth. like below could help avoid this issue.
```
WithSQLConf("spark.sql.orc.enableVectorizedReader" -> "false") {
  content
}
```

cc @zhztheplayer

Reply:
@rui-mo 
Thank you for your detailed explanation. It seems that these three configurations cannot take effect. Because the test unit test is aim to test the columnar batch scan [here](https://github.com/apache/spark/blob/af50b47e12040f86c4f81ff84407ad820cb252c1/sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2Suite.scala#L209).